EMacco/GoBanka
# #168912734 User Login

#### What does this PR do?
This PR allows a user to be able to login

#### Description of Task to be completed?
- When user is logged in, he should be redirected to the dashboard
- Three failed attempts should deactivate the user account

#### How should this be manually tested?
- Checkout to this branch
- Attempt to login
- If details are correct you should be redirected to the home page 

#### What are the relevant pivotal tracker stories?
[#168912734](https://www.pivotaltracker.com/story/show/168912734)

#### Screenshots (if appropriate)
<img width="399" alt="Screenshot 2019-10-03 at 2 39 36 PM" src="https://user-images.githubusercontent.com/20377428/66131608-a502a500-e5eb-11e9-91a8-bcd195f48771.png">
